Desktop/html/css/printing.css

```diff
@@ -51,16 +51,17 @@
      .jasp-table-primitive {
          page-break-inside: avoid;
     }
-     .jasp-table table {
+    /* .jasp-table table {
          table-layout:fixed !important;
-         width: 20cm !important;
-    }
+         width: 100% !important;
```

Comment:
This css obliged the table to be 20 cm wide (even if the table is self much smaller). This allows table that are wider than 20 cm to fit in a A4 format, but most of the time it is not a valid solution since the table is squeezed, and the columns overlapped each other.
To set it explicitly to 20cm does not work anyway for landscape print (or for other page size).
If I set the the width to 100% (with the table-layout fixed), then it fits the landscape size, but then all tables are as wide as the whole width of the page.
Unfortunately, the max-width does not work with table-layout fixed, this would be in fact what we are looking for.
As I cannot see a right solution here, I prefer to remove this setting.
I find the argument that the cut-off might not be noticed by users, a bit poor compared to the problems this setting generate. I find we see quite easily that a part of a table misses when it is too big.

Reply:
Yes, set 20cm explicitly is only for A4.

Reply:
We could set the exact size for each different combination of pagesize + orientation. But I wouldnt want to hold up the PR for that :p